larpandora/LArPandoraInterface/LArPandoraGeometry.h

```diff
@@ -381,6 +387,13 @@ inline float LArDetectorGap::GetZ2() const
     return m_z2;
 }
 
+//------------------------------------------------------------------------------------------------------------------------------------------
+
+inline float LArDetectorGap::GetMaxGapSize()
```

Comment:
Recommend decorating this with `noexcept` as it cannot throw an exception.  Will need to make corresponding change to function declaration.